python/inspector.in

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
How about

```
#! @PYTHON_EXECUTABLE@
```

instead?

This variable get set by `find_package(PythonInterp 3 REQUIRED)` in `python/CMakeLists.txt`.
Then it depends only at build time on the user's PATH.

Reply:
I heard that `#!/usr/bin/env python3` should be preferred over `#!/usr/bin/python3`. Can't find any more details about it though.

I would say keep it simple and don't rely on CMake. This would allow to change python3's path after building.

Reply:
I do not consider changing python3 as a problem - it makes the build more reliable. However cmake seems to not pick up virtualenv, which people might rely on to install dependencies. 
Most system package manager will have some scripts to replace `/usr/bin/env` on installation, so this pr should be fine.